### ISSUE TITLE
Stagger order replacement

### ISSUE THREAD

**Issue by lomacks** [lomacks/bittrex-order-refresh#3]
Instead of replacing any orders older than (default) 14 days at once, we could stagger them, e.g. make the threshold 14 +/- random(5) days. This would cause order age to "spread out" over time so that individual script runs have less work to do in future.

**Comment by michalbirecki** [comment]
Good idea -- what's the downside of running it daily or every let's say 12 hours though?

**Comment by lomacks** [comment]
Yeah, there should be no problem running it at that sort of frequency, except that in its current state it preserves "hotspots" of heavy activity - say e.g. you placed 100 orders one day, and none for the next couple of weeks, that pattern would repeat every 14 days, with no work to do on the days in between.

Evening out the load over time might tangentially help out issues like #6, though since that will be handled by other means, it's probably more of a nice to have.

**Comment by lomacks** [comment]
(The point being that a smaller, random number of orders per day is less likely to hit API limits etc. than doing many at once on occasional runs.)
